samples/ingestion/ingestion-client/FetchTranscription/Program.cs

```diff
@@ -0,0 +1,44 @@
+// <copyright file="Program.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
+// </copyright>
+
+namespace FetchTranscription
+{
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.ApplicationInsights.WorkerService;
+    using Microsoft.Azure.Functions.Worker;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.Extensions.Hosting;
+    using Microsoft.Extensions.Logging;
+
+    public static class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            var host = new HostBuilder()
+                .ConfigureFunctionsWorkerDefaults()
```

Comment:
>ConfigureFunctionsWorkerDefaults [](http://example.com/codeflow?start=17&length=32)

looking at the guide we should use ConfigureFunctionsWebApplication since we're targeting asp net core:

[https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide?tabs=linux](https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide?tabs=linux)

'Call either ConfigureFunctionsWebApplication() if using ASP.NET Core integration or ConfigureFunctionsWorkerDefaults() otherwise.'


we don't care about .net framework :)

Reply:
Reading this section [https://learn.microsoft.com/en-us/azure/azure-functions/migrate-dotnet-to-isolated-model?tabs=net8#programcs-file](https://learn.microsoft.com/en-us/azure/azure-functions/migrate-dotnet-to-isolated-model?tabs=net8#programcs-file) it says if we are using HttpTriggers then use ConfigureFunctionsWebApplication - otherwise use ConfigureFunctionsWorkerDefaults. The Triggers are Timers and ServiceBus

What are your thoughts?

Reply:
ah makes sense, didn't see that!